src/objects.rs

```diff
@@ -17,30 +17,33 @@ use std::ops::{Deref, DerefMut};
 /// tags](http://wiki.openstreetmap.org/wiki/Tags) for more
 /// information.
 #[derive(Debug, Default, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
-pub struct Tags(TagsImpl);
-/// FlatMap representing the key-value pairs of the tags
-pub type TagsImpl = ::flat_map::FlatMap<String, String>;
+pub struct Tags(::flat_map::FlatMap<String, String>);
```

Comment:
Maybe just not using the type alias here would be enough to have the documentation. Looks like a bug, no?

Reply:
yep: https://github.com/rust-lang/rust/issues/35295

Reply:
https://github.com/rust-lang/rust/issues/68016

And the alias is useless, unless I'm missing something?

Reply:
The alias is more here for backcompatibility, before, this TagsImpl was a new type if I recall well.